packages/typegpu/src/data/atomic.ts

```diff
@@ -27,11 +27,14 @@ class AtomicImpl<TSchema extends U32 | I32>
 {
   public readonly size: number;
   public readonly byteAlignment: number;
+  isLoose = false as const;
+  isCustomAligned: boolean;
 
   constructor(private readonly innerData: TSchema) {
     super();
     this.size = this.innerData.size;
     this.byteAlignment = this.innerData.byteAlignment;
+    this.isCustomAligned = innerData.isCustomAligned;
```

Comment:
Tiny nitpick, for consistency
```suggestion
    this.isCustomAligned = this.innerData.isCustomAligned;
```